src/THighlighter.cpp

```diff
@@ -22,136 +22,142 @@
+    keywordFormat.setForeground(Qt::darkBlue);
+    keywordFormat.setFontWeight(QFont::Bold);
     QStringList keywordPatterns;
-    keywordPatterns << "\\btrue\\b" << "\\bfalse\\b" << "\\bnil\\b"
-                    << "\\bnot\\b" << "\\band\\b" << "\\bor\\b"
-                    << "\\bfunction\\b" << "\\blocal\\b" << "\\end\\b"
-                    << "\\bwhile\\b" << "\\bdo\\b" << "\\bif\\b"
-                    << "\\bthen\\b" << "\\belse\\b" << "\\bwhile\\b"
-                    << "\\brepeat\\b" << "\\bfor\\b" << "\\bpairs\\b"
-                    << "\\bipairs\\b" << "\\bin\\b" << "\\buntil\\b"
-                    << "\\bbreak\\b" << "\\breturn\\b" << "\\belseif\\b";
-    foreach( QString pattern, keywordPatterns )
-    {
-        rule.pattern = QRegExp( pattern );
+    keywordPatterns << "\\btrue\\b"
+                    << "\\bfalse\\b"
+                    << "\\bnil\\b"
+                    << "\\bnot\\b"
+                    << "\\band\\b"
+                    << "\\bor\\b"
+                    << "\\bfunction\\b"
+                    << "\\blocal\\b"
+                    << "\\end\\b"
+                    << "\\bwhile\\b"
+                    << "\\bdo\\b"
+                    << "\\bif\\b"
+                    << "\\bthen\\b"
+                    << "\\belse\\b"
+                    << "\\bwhile\\b"
+                    << "\\brepeat\\b"
+                    << "\\bfor\\b"
+                    << "\\bpairs\\b"
+                    << "\\bipairs\\b"
+                    << "\\bin\\b"
+                    << "\\buntil\\b"
+                    << "\\bbreak\\b"
+                    << "\\breturn\\b"
+                    << "\\belseif\\b";
+    foreach (QString pattern, keywordPatterns) {
+        rule.pattern = QRegExp(pattern);
         rule.format = keywordFormat;
-        highlightingRules.append( rule );
+        highlightingRules.append(rule);
     }
 
 
-
-    classFormat.setFontWeight( QFont::Bold );
-    classFormat.setForeground( Qt::darkMagenta );
-    rule.pattern = QRegExp( "\\bQ[A-Za-z]+\\b" );
+    classFormat.setFontWeight(QFont::Bold);
+    classFormat.setForeground(Qt::darkMagenta);
+    rule.pattern = QRegExp("\\bQ[A-Za-z]+\\b");
     rule.format = classFormat;
-    highlightingRules.append( rule );
+    highlightingRules.append(rule);
 
-    functionFormat.setFontItalic( false );
-    functionFormat.setFontWeight( QFont::Bold );
-    functionFormat.setForeground( Qt::black );
-    rule.pattern = QRegExp( "\\b[A-Za-z0-9_]+(?=\\()" );
+    functionFormat.setFontItalic(false);
+    functionFormat.setFontWeight(QFont::Bold);
+    functionFormat.setForeground(Qt::black);
+    rule.pattern = QRegExp("\\b[A-Za-z0-9_]+(?=\\()");
     rule.format = functionFormat;
-    highlightingRules.append( rule );
+    highlightingRules.append(rule);
 
-    quotationFormat.setForeground( Qt::darkGreen );
-    rule.pattern = QRegExp( "\"[^\"]*\"" );
+    quotationFormat.setForeground(Qt::darkGreen);
```

Comment:
:thinking: That reminds me, quotation formatting breaks for escaped quotes in the Lua Script editor, i.e. a literal `\"` is actually treated like a plain `"` when one wants to embed " in a string to display to the user...